demos/gpu/README.md

```diff
@@ -12,7 +12,7 @@
 The GPU demos are aimed at demonstrating how to run code over graphics processing units (GPUs) in the Iguazio Data Science Platform (**the platform**).
 The **demos/gpu** directory includes the following:
 
-- A **horovod** directory with applications that use Uber's [Horovod](https://eng.uber.com/horovod/) distributed deep-learning framework, which can be used to convert a single-GPU TensorFlow, Keras, or PyTorch model-training program to a distributed program that trains the model simultaneously over multiple GPUs.
+- A **horovod** directory with applications that use Uber's [Horovod](https://eng.uber.com/horovod/) [(github)](https://github.com/horovod/horovod) distributed deep-learning framework, which can be used to convert a single-GPU TensorFlow, Keras, or PyTorch model-training program to a distributed program that trains the model simultaneously over multiple GPUs.
     The objective is to speed up your model training with minimal changes to your existing single-GPU code and without complicating the execution.
     Horovod code can also run over CPUs with only minor modifications.
```

Comment:
I recommend keeping lines 15–17 as-is and adding the following:
```
For more information and examples, see the [Horovod GitHub repository](https://github.com/horovod/horovod).
```
[HTML output — "For more information and examples, see the [Horovod GitHub repository](https://github.com/horovod/horovod)."]

Reply:
Approved

Reply:
UPDATE: I implemented the suggested changes in PR #152.